Controllers/AdminController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Controllers;
+
+class AdminController  extends BasicController
```

Comment:
Apply coding standards: two spaces before extends.
Please use Ctrl + Alt + L before push. It will automatically remove extra white spaces.